tests/utils.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals, absolute_import
+
+from datetime import datetime
+
+from estuary.models.koji import KojiBuild
+import koji
+import pytz
+
+
+def mock_getBuild(state=koji.BUILD_STATES['COMPLETE'], isCommit=False):
+    """
+    Make a mock set of build data simulating a Koji getBuild API call.
+
+    :param int state: the state of the build
+    :param bool isCommit: true if a commit is connected, false otherwise
+    :return: a dictionary with all of the build info
+    :rtype: dict
+    """
+    if state == koji.BUILD_STATES['COMPLETE']:
+        completion_time = '2018-06-15 20:26:38.000000'
+        completion_ts = 1529094398.0
+    else:
+        completion_time = None
+        completion_ts = None
+
+    if isCommit:
+        extra = {'source': {'original_url': 'git://pkgs.domain.com/rpms/squashf'
+                 's-tools#09d40c9bdfd34c5130f8f02e49e059efd33bddf7'}}
+    else:
+        extra = {'container_koji_task_id': 17511743}
+
+    return {
+        'completion_time': completion_time,
+        'completion_ts': completion_ts,
+        'creation_time': '2018-06-15 20:20:38.000000',
+        'creation_ts': 1529094038.0,
+        'epoch': 'epoch',
+        'extra': extra,
+        'id': 710916,
+        'name': 'e2e-container-test-product-container',
```

Comment:
All the test messages you added don't match this mock build you are returning. You should modify the messages to match this because the tests are now actually changing more values than just the state.